### PR TITLE
REVIEW THE PAST PROJECT

### PR DESCRIPTION
**Pull request by pmaxy**
Things done:

- Run linters 
- Add gitignore file'
- Add html and css files
-

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      <br>
+      <h4>The Downsides of AI artistry</h4>
+      <p>What are possible adverse effects of on-demand Ai image generation?</p>
+      <br>
+      <hr>
+      <br>
+      <h4>Is VC Funding Drying Up?</h4>
+      <p>Private funding by VC films of down 50% YOY. We take a look at the means </p>   
+    </div>
+    <div class="thd">
```
Comment (pmaxy) [inline index.html:45]:
```suggestion
    <div class="thd">
```

**Review by pmaxy — COMMENTED** [review]
HELLO @pmaxy  👍 

changes
Very amazing and good job. keep it with the good work. Kindly do the following last changes before the approval:
:Rename the class and make it professional  😄 



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct Github flow ✔️ 
- Nice UI design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  <img src="images/logo.svg" alt="">
+  </div>
+  <div class="navbar">
+    <ul>
+      <li class="item"><a href="#">Home</a></li>
+      <li class="item"><a href="#">New</a></li>
+      <li class="item"><a href="#">Popular</a></li>
+      <li class="item"><a href="#">Trending</a</li>
+      <li class="item"><a href="#">Cartegories</a></li>
+    </ul>
```
Comment (pmaxy) [inline index.html:22]:
```suggestion
    <ul>
      <li class="item"><a href="#">Home</a></li>
      <li class="item"><a href="#">New</a></li>
      <li class="item"><a href="#">Popular</a></li>
      <li class="item"><a href="#">Trending</a</li>
      <li class="item"><a href="#">Cartegories</a></li>
    </ul>
```

**Review by pmaxy — COMMENTED** [review]
Good work on this session but kindly, Make the links below responsive and insert their link.

File: index.html (hunk 2 above)
Comment (TadesseDev) [inline index.html:22]: - [ ] Dear @pmaxy, Good job submitting a suggestion, But simply adding a block of code does not tell what is wrong with the current implementation and why you are asking for the changes. Please make sure to explain `why` you ask for a change in a `polite`, `clear `, and `easy to understand` way.

File: index.html (hunk 1 above)
Comment (TadesseDev) [inline index.html:45]: - [ ] Dear @pmaxy, Good job submitting a suggestion, But simply adding a block of code does not tell what is wrong with the current implementation and why you are asking for the changes. Please make sure to explain `why` you ask for a change in a `polite`, `clear `, and `easy to understand` way.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @pmaxy 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7

## Highlights
 - [ ]  You have submitted a review in the Pull Request you opened  ✅
 - [ ]  You have used a GIF to make your review more engaging  ✅
 
## Required Changes ♻️
- [ ] Dear @pmaxy, One of the major requirements for this milestone is to write at least 2 comments in the review. I can see you have two independent review's each with a single comment. Please make sure to submit a single review with two inline comments.
- [ ] Dear @pmaxy, Good job submitting a suggestion, I see your comments and you don't have an explanation added to your inline comments, Simply adding a block of code does not tell what is wrong with the current implementation and why you are asking for the changes. Please make sure to explain `why` you ask for a change in a `polite`, `clear `, and `easy to understand` way.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (pmaxy) [inline index.html:22]: kindly ensure that you link the following lists for the responsiveness
```suggestion
    <ul>
      <li class="item"><a href="#">Home</a></li>
      <li class="item"><a href="#">New</a></li>
      <li class="item"><a href="#">Popular</a></li>
      <li class="item"><a href="#">Trending</a</li>
      <li class="item"><a href="#">Cartegories</a></li>
    </ul>
    

```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      <h4>Is VC Funding Drying Up?</h4>
+      <p>Private funding by VC films of down 50% YOY. We take a look at the means </p>   
+    </div>
+    <div class="thd">
+      <h1>The Bright Future of Web 3.0?</h1>
+    </div>
+    <div class="fth">
+      <p>We dive into the evolution of the web that claims to
+          put the power of the platforms back into the hands of the people.
+          But is it really fulfilling its promise?
```
Comment (pmaxy) [inline index.html:51]: Kindly make sure that classes are named professionally example fifth_content.
```suggestion
    <div class="fth">
      <p>We dive into the evolution of the web that claims to
          put the power of the platforms back into the hands of the people.
          But is it really fulfilling its promise?
```

**Review by pmaxy — COMMENTED** [review]
HELLO @pmaxy  👍 

:Rename the class and make it professional  😄 



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Correct Github flow ✔️ 
- Nice UI design ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
Hi @pmaxy  👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so_close


 ## Highlights 🎯 
- Great commit messages ✅ 
- Impressive effort ✅ 

## Required Changes ♻️
- [x] I have gone through the first review that my colleague gave [here](https://github.com/pmaxy/Review_past_project/pull/1#pullrequestreview-1251686623), and I think you should read it through again, also try to check this review, so that you can notice the communication link. It's important to write reviews as though you are talking because it ensures clarity.
- [ ] Your style of expression is awesome but some parts of the constructs might be a bit too direct or lack clarity, and as a code reviewer, you want to deploy your empathy to making sure that the recipient learn by telling them why your suggestion is better, become better rather than trying command them. Making them feel comfortable will increase your chance of being a help to them.

- Good reviews that allow recipients to learn have 4 major things: `what the problem is`, `why it is an issue`, `how to resolve the issue` and `reasons for the proposition`. e.g: Your comment could have been something like:
```
Kindly please set up your linter environment properly as it is part of the project requirement. Currently, I noticed your stylelint is not passing at the moment. Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linters properly, and try to make necessary corrections just to be sure that all the tests passed i.e marked checked with green or blue color. 
This important because it's an established coding standard in real work environment to ensure uniformity of code to avoid potential bugs.
```
_Isn't that a little better than the single statements you included earlier? I think you should think of it like you are talking to a friend who needs your advice. Be natural, and communicate with clarity but with empathy too._
- [ ] Good point with some of the issues you enumerated but it appears you forgot to point to the writer where the issues are and where he could have made corrections in the code using inline comments, and another thing you could do to help is to provide images of the current design as evidence and to form a basis for your suggestion like this 👇 (from your review):

image
**_Like the previous CRs indicated, communicating with peers, colleagues, or juniors on their code would have more goodly effect if the choice of words are `polite`, `kind`, not trying to command or force your own way of doing things_**

👉 👉  _**Here's a link to a short tutorial on how to use [GitHub inline comments](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request)**_ to make your reviews look a like a pro.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- You can also consider making your summary more engaging and inspiring. Let him/know what was done perfectly, and that he is almost done if you had to request for change.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,157 @@
-# Review_past_project
\ No newline at end of file
+# Portfolio-setup-and-mobile-first<a name="readme-top"></a>
+
```
Comment (pmaxy) [inline README.md:2]: Kindly,  I think the heading should match the topic, As per the guideline from Microverse
```suggestion
# Portfolio-setup-and-mobile-first<a name="readme-top"></a>

```

File: index.html (hunk 2 above)
Comment (pmaxy) [inline index.html:22]: Kindly, i think the links should be responsive, for the project to be professional
```suggestion
    <ul>
      <li class="item"><a href="#">Home</a></li>
      <li class="item"><a href="#">New</a></li>
      <li class="item"><a href="#">Popular</a></li>
      <li class="item"><a href="#">Trending</a</li>
      <li class="item"><a href="#">Cartegories</a></li>
    </ul>
```

**Review by pmaxy — COMMENTED** [review]
hello @pmaxy

Great job so far! 👏
You've done it well

# **To highlight** 🎯

- [ ] - No linter errors.✔️
- [ ] - Nice design. ✔️
- [ ] - Descriptive PR. ✔️

unnamed

## Required Changes ♻️

-   Good points and awesome design by this far , continue with the good work . Kindly I think on the readme topic should match the project topic, this will make your project look good and one can be able to identify the project you are working on.
- Also you can make the links on navbar be responsive to make it look nice  by giving the links for the specific address.

 

 
# Optional suggestions

- N/A

**_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

<hr>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).<hr>

**Review by IngDavidTM — APPROVED** [review]
Hi @pmaxy 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- Your comments are polite ✅  


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**